bbq/vm/vm.go

```diff
@@ -837,7 +837,9 @@ func opRemoveIndex(vm *VM) {
 		EmptyLocationRange,
 		index,
 	)
-	containerValue.SetKey(
+
+	// Note: Must use `InsertKey` here, not `SetKey`.
+	containerValue.InsertKey(
```

Comment:
What is the difference between insert and set, and why is it important to use insert?